<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" tests="10" skipped="0" failures="0" errors="0" timestamp="2023-07-06T21:57:43" hostname="fv-az336-32" time="0.223">
  <properties/>
  <testcase name="testEmptyParameter" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.16"/>
  <testcase name="testUserPasswordException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.035"/>
  <testcase name="testgetUsersWithLoginAs" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.005"/>
  <testcase name="testAuthException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testgetUsersWithoutLoginAs" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testNoSuchUserException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testRequiredLayoutException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testUserActiveException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.002"/>
  <testcase name="testWrongParameter" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.001"/>
  <testcase name="testUserEmailAddressException" classname="com.dotcms.rest.api.v1.authentication.AuthenticationResourceTest" time="0.008"/>
  <system-out><![CDATA[ErrorEntity{errorCode='your-account-is-not-active', message='your-account-is-not-active', fieldName='null'}
]]></system-out>
  <system-err><![CDATA[Jul 06, 2023 9:57:43 PM com.dotcms.repackage.org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 4.3.2.Final
]]></system-err>
</testsuite>
